foxy/ci-nightly-cross-vendor-connext-cyclonedds.yaml

```diff
@@ -14,6 +14,7 @@ install_packages:
 - default-jdk # for CycloneDDS
 - libtinyxml2-dev # for FastRTPS, even though disabled it is needed for the typesupport
 - maven # for CycloneDDS
+- libssl-dev  # for Connext and CycloneDDS
```

Comment:
Nitpick: please insert in alphabetical order.

Same below.

Reply:
:+1: Done. b348375